include/scrimmage/plugins/motion/JSBSimModel/JSBSimModel.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0"?>
 <?xml-stylesheet type="text/xsl" href="http://gtri.gatech.edu"?>
 <params>
-  <library>JSBSimModel_plugin</library>  
+  <library>JSBSimModel_plugin</library>
   <use_pitch>false</use_pitch>
 
+  <jsbsim_root><jsbsim_root>
```

Comment:
Should this be: <jsbsim_root></jsbsim_root>

Reply:
Fixed and repushed the commit